docs/ABOUT.md

```diff
@@ -0,0 +1,14 @@
+Javascript is a scripting language used to provide dynamic and interactive content on webpages. It does not have class-oriented inheritance, but it does have prototype-oriented inheritance. JavaScript provides benefits like :- 
+* Less server interaction as in validating user input before sending page to server, thus, reducing server traffic.
```

Comment:
This should probably read:
'JavaScript in the browser can reduce server traffic by validating user input before it is sent to the server.'